GLTFSDK/Source/Deserialize.cpp

```diff
@@ -413,13 +413,33 @@ namespace
                 throw InvalidGLTFException("Camera perspective projection undefined");
             }
 
-            float aspectRatio = GetMemberValueOrDefault<float>(perspectiveIt->value, "aspectRatio", 0.0f);
+            Optional<float> aspectRatio;
+
+            auto itAspectRatio = perspectiveIt->value.FindMember("target");
+            if (itAspectRatio != perspectiveIt->value.MemberEnd())
+            {
+                aspectRatio = itAspectRatio->value.GetFloat();
+            }
+
             float yfov = GetValue<float>(FindRequiredMember("yfov", perspectiveIt->value)->value);
             float znear = GetValue<float>(FindRequiredMember("znear", perspectiveIt->value)->value);
-            float zfar = GetMemberValueOrDefault<float>(perspectiveIt->value, "zfar", std::numeric_limits<float>::infinity());
-            projection = std::make_unique<Perspective>(zfar, znear, aspectRatio, yfov);
 
-            ParseProperty(perspectiveIt->value, *projection, extensionDeserializer);
+            Optional<float> zfar;
+
+            auto itZFar = perspectiveIt->value.FindMember("zfar");
+            if (itZFar != perspectiveIt->value.MemberEnd())
+            {
+                zfar = itZFar->value.GetFloat();
+            }
+
+            auto perspective = std::make_unique<Perspective>(znear, yfov);
+
+            perspective->zfar = zfar;
```

Comment:
std::move(zfar), aspectratio?

Reply:
There is no advantage to moving (rather than copying) primitive types (such as float)